eng/versioning/version_client.txt

```diff
@@ -3,7 +3,7 @@
 
 com.azure:azure-sdk-parent;1.6.0;1.6.0
 com.azure:azure-client-sdk-parent;1.7.0;1.7.0
-com.azure:azure-core;1.1.0;1.1.0
+com.azure:azure-core;1.1.1;1.1.1
```

Comment:
I'm going to approve the changes but I do have a question. Did we need to update both the _dependency_ and _current_ versions or should the _dependency_ version still be 1.1.0?